Sources/Data/Microsoft.Psi.Data/Dataset.cs

```diff
@@ -399,6 +469,8 @@ public Session AddSessionFromStore(IStreamReader streamReader, string sessionNam
                     progress != null ? new Progress<(string, double)>(tuple => progress.Report((tuple.Item1, (sessionStart[i] + tuple.Item2 * sessionDuration[i]) / totalDuration))) : null,
                     cancellationToken);
             }
+
+            this.OnDatasetChanged();

```

Comment:
Same here, is this required, or will the session.CreateDerivedPsiPartitionAsync trigger the event?